File: UI/Filp_Coin_Factor/Filp_Coin_Factor/ViewController.m
//
//  ViewController.m
//  Filp_Coin_Factor
//
//  Created by gaurav on 02/03/17.
//  Copyright © 2017 gaurav. All rights reserved.
//

#import "ViewController.h"
#import "math.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)btnFlip:(id)sender {
    NSString *a=_txtflip.text;
    int b=[a intValue];
    
    int count = 1;
    double result;
    int h=0,t=0;
    while(count < b)
    {
        result = rand()/(double)RAND_MAX;;
        if(result <= 0.5)
            h++;
        else
            t++;
        
        count++;
    }
    float headper=(h/b)*100;
    float tailper=100-headper;
   
    [self.lblHeadper setText:[NSString stringWithFormat:@"%0.2f",headper]];
    
    [self.lblTailPer setText:[NSString stringWithFormat:@"%0.2f",tailper]];
    
}
- (IBAction)btnFactor:(id)sender {
    NSString *factor =_txtFactor.text;
    int fct=[factor intValue];
    int i = 1;
    NSMutableArray *ab=[[NSMutableArray alloc]init];
    
    while(i<=fct)
    {
        if(fct%i == 0)
        [ab addObject:[NSString stringWithFormat:@"%d",i]];
       // [self.lblFactor setText:[NSString stringWithFormat:@"%d,",i]];
        i++;
    }
   // NSString *str =[NSString stringWithFormat:@"%@",ab];
   // 	[self.lblFactor setText:str];
    NSString *greeting = [ab componentsJoinedByString:@", "];

    [self.lblFactor setText:[NSString stringWithFormat:@"%@,",greeting]];
    //NSLog(@"dfgdfdf%@",str);
    
}

@end

File: Swift/TableView1/TableView1/ViewController.swift
//
//  ViewController.swift
//  TableView1
//
//  Created by gaurav on 14/03/17.
//  Copyright © 2017 Alk. All rights reserved.
//

import UIKit

class ViewController: UIViewController,UITableViewDelegate,UITableViewDataSource {
    let list=["A","B","c","d","e","f"];
    @available(iOS 2.0, *)
    public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int
    {
        return(list.count);
    }
    @available(iOS 2.0, *)
    public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell
    {
        let cell=UITableViewCell(style:UITableViewCellStyle.default,reuseIdentifier:"cell");
        cell.textLabel?.text=list[indexPath.row];
        return(cell);
    }

    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    
  }

